Lesson 4/Lesson 4/Lesson 4.cs

```diff
@@ -0,0 +1,236 @@
+﻿using System;
+
+Console.WriteLine("Write the number of elements in the array and press Enter:");
+var userInput = Console.ReadLine();
+int arrayLength = CheckUserInput(userInput);
+
+int[] numbers = GenerateArray(arrayLength);
+Console.WriteLine("Generated array:");
+DisplayNumericArray(numbers);
+
+int[] evenNumbers = GenerateArrayWithEvenNumbers(numbers);
+Console.WriteLine();
+Console.WriteLine("Array with even numbers:");
+DisplayNumericArray(evenNumbers);
+
+int[] oddNumbers = GenerateArrayWithOddNumbers(numbers);
+Console.WriteLine();
+Console.WriteLine("Array with odd numbers:");
+DisplayNumericArray(oddNumbers);
```

Comment:
We can create one method instead of these two GenerateArrayWithEvenNumbers, GenerateArrayWithOddNumbers . In this case, we will need to create method that will accept type of numbers that we need either odd or even